.github/workflows/linters.yml

```diff
@@ -0,0 +1,44 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/233622605-a37d1d44-36fe-40d6-99f4-4c009df779ed.png)

**[OPTIONAL]**

- [ ] It would be nice if you make your commit messages more meaningful and capitalize 👍 because some of your commit messages are not meaningful enough 😨 Please consider this issue to make your pull request professional and descriptive ☺️